### PR TITLE
[+] basic auth

### PR DESCRIPTION
**Pull request by lbchv**
#6 basic auth from environment variables

### CODE REVIEW

**Comment by ageis** [comment]
@pezzak I'm not 100% on this (obtained the advice from a generic Go HTTP client w/ basic auth tutorial), but I think there's a possibility you might want a handler to ensure the authorization header is not lost on redirects. for example...

before main():
```
func basicAuth(username, password string) string {
	auth := username + ":" + password
	return base64.StdEncoding.EncodeToString([]byte(auth))
}

func redirectPolicy(req *http.Request, via []*http.Request) error{
    req.Header.Add("Authorization","Basic " + basicAuth(esUser,esPass))
    return nil
}
```
in handler() before `client.Do(req)`:
```
client := &http.Client{
	    CheckRedirect: redirectPolicy,
}
```

File: main.go [hunk 1]
```diff
@@ -64,7 +64,7 @@ func main() {
 	flag.StringVar(&esURL, "esURL", esURL, "Elasticsearch HTTP URL")
 	flag.BoolVar(&showVersion, "version", false, "Print version number and exit")
 	flag.Parse()
-
+
```
Comment (sinkingpoint) [inline main.go:67]: Nit: Extra whitespace here

**Comment by sinkingpoint** [comment]
@pezzak Thanks for the PR! If you fix up that one nit (And address Kevin's comment) I'd be happy to merge this

**Comment by s-starostin** [comment]
Hello, guys, I forked pezzak's repo and made required fixes. Also I added -disableCertCheck option for alertmanager2es. Could you please review and merge it?
https://github.com/cloudflare/alertmanager2es/pull/17

**Comment by sinkingpoint** [comment]
Closing this as being included in #17

**Pull request closed without merging**